engine/scripts.py

```diff
@@ -7,8 +7,10 @@
     Protocol,
     Type,
     Tuple,
+    Iterable,
 )
 
+from arcade import Sprite
```

Comment:
Ditto on the style nit. In this case since `arcade` isn't imported already, just `import arcade`.

Reply:
Done